ui/apps/platform/src/Containers/Vulnerabilities/utils/sortUtils.tsx

```diff
@@ -21,10 +29,21 @@ export const aggregateByCreatedTime: SortAggregate = {
  * @param entityTab The chosen entity
  * @returns The available sort fields
  */
-export function getWorkloadSortFields(entityTab: WorkloadEntityTab): string[] {
+export function getWorkloadSortFields(entityTab: WorkloadEntityTab): (string | string[])[] {
     switch (entityTab) {
         case 'CVE':
-            return ['CVE', 'CVSS', 'Image Sha', 'CVE Created Time'];
+            return [
+                'CVE',
+                [
+                    'Critical Severity Count',
+                    'Important Severity Count',
+                    'Moderate Severity Count',
+                    'Low Severity Count',
+                ],
+                'CVSS',
+                'Image Sha',
+                'CVE Created Time',
+            ];
```

Comment:
Image array below caught my eye. Do `'Image created time'` and `'Image scan time'` correspond:
https://github.com/stackrox/stackrox/blob/master/pkg/search/options.go#L80
https://github.com/stackrox/stackrox/blob/master/pkg/search/options.go#L87

Reply:
Find in Files with regexp found only 2 others:
* `'Image sha'` in RequestCVEsTable.tsx
* `'Images by severity'` in (comment of) useURLSort.ts file

Reply:
It is a good time for some consistency, updated!